src/explorer/resultsList.ts

```diff
@@ -748,4 +732,64 @@ export class ResultsList {
             caseButton.classList.remove("active");
         }
     }
+
+    /**
+     * Handler to perform keyboard navigation in the results list.
+     * @param keydownEvent Keyboard event
+     */
+    private onResultsListKeyDown(keydownEvent: JQuery.KeyDownEvent): void  {        
+        enum KeyCodes {
+            LeftArrow = 37,
+            UpArrow = 38,
+            RightArrow = 39,
+            DownArrow = 40
+        }
+
+        let direction: 'Previous' | 'Next' | undefined;
+        if (keydownEvent.keyCode === KeyCodes.LeftArrow || keydownEvent.keyCode === KeyCodes.UpArrow) {
+            direction = 'Previous';
+            
+        } else if (keydownEvent.keyCode === KeyCodes.RightArrow || keydownEvent.keyCode == KeyCodes.DownArrow) {
+            direction = 'Next';
+        }
+
+        if (!direction) {
+            return;
+        }
+
+        const curSelected: HTMLCollectionOf<Element> | undefined = document.getElementsByClassName("listtablerow selected");
+        if (!curSelected || curSelected.length !== 1) {
+            return;
+        }
+
+        let currentSelectedElement: Element | undefined = curSelected[0];
+        while (currentSelectedElement) {
+            const potentialElementToSelect: Element | null = direction === 'Next' ? currentSelectedElement.nextElementSibling : currentSelectedElement.previousElementSibling;
```

Comment:
>null [](start = 54, length = 4)

just making sure - is this actually `null` or should it be `undefined`?

Reply:
Yes, it really is null.

---
In reply to: [413112674](https://github.com/microsoft/sarif-vscode-extension/pull/254#discussion_r413112674) [](ancestors = 413112674)